### PR TITLE
Update jest-utils

### PR DESCRIPTION
**Pull request by viceice**
Update `jest-utils` to `^25.1.0`

### CODE REVIEW

**Comment by SimenB** [comment]
Should probably bump to 26.

/cc @rickhanlonii

**Comment by rickhanlonii** [comment]
Yeah could you bump this to 26?

**Comment by viceice** [comment]
done 🙃

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
   },
   "dependencies": {
     "chalk": "^2.3.1",
```
Comment (SimenB) [inline package.json]: v4? 😀 that's what's used internally by jest@26

Comment (viceice) [reply package.json]: i can bumb chalk too

**Comment by viceice** [comment]
also updated chalk to v4

**Review by SimenB — APPROVED** [review]
fwiw

**Pull request merged**